src/Cake.Testing/FakeEnvironment.cs

```diff
@@ -109,6 +111,12 @@ public string GetEnvironmentVariable(string variable)
             return new Dictionary<string, string>(_environmentVariables, StringComparer.OrdinalIgnoreCase);
         }
 
+        /// <inheritdoc/>
+        public IDictionary<string, string> GetArguments()
```

Comment:
This will be available in the fixtures (might need tweaking a fixture or create a new one)

Reply:
Checked, that `CakeContextFixture` has `Arguments = Substitute.For<ICakeArguments>();` line. Do I need to add a new Fixture or some tests with that?